src/overlay/test/OverlayTestUtils.h

```diff
@@ -0,0 +1,25 @@
+#pragma once
+
+// Copyright 2022 Stellar Development Foundation and contributors. Licensed
+// under the Apache License, Version 2.0. See the COPYING file at the root
+// of this distribution or at http://www.apache.org/licenses/LICENSE-2.0
+
+#include "main/Application.h"
+
+namespace stellar
+{
+
+std::uint64_t getOverlayFloodMessageCount(std::shared_ptr<Application> app,
```

Comment:
use uint64_t for consistency

Reply:
also, `getOverlayFloodMessageCount` doesn't need to accessible outside of overlay test utils

Reply:
Done